templates/f5/0/docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+external-lb:
+  image: rancher/external-lb:v0.1.1
+  command: -debug -provider=f5_BigIP
```

Comment:
I imagine this is really in the image... but underscores, lowercase `f`, capital `B` `IP`?